server/service/taskValidator.js

```diff
@@ -0,0 +1,14 @@
+const taskModel = require('../model/taskModel');
+const { NotFoundError, UnauthorizedError } = require('../errorTypes/errorTypes')
+module.exports = {
+    validateUserTask: async (id, user) => {
```

Comment:
this can be stored as middleware

Reply:
Okay i store it in the middleware folder but it should be used like a normal function, not like a middleware, right?